packages/ffe-form/less/phone-number.less

```diff
@@ -1,5 +1,11 @@
 .ffe-phone-number {
     &__input-group {
+        @media (min-width: @breakpoint-sm) {
```

Comment:
burde vi bruke container queries istellet for media queries når vi liks er i gang. Det har veldig fint browserstøtte. Hvis man har 3 kolumner på desktop feks.

Reply:
Det kan godt være fornuftigere ja. Men kanskje vi kan gjøre den forbedringen i en ny PR, så vi kan se på om breakpointene fortsatt gir mening i den konteksten